spec/spec_helper.rb

```diff
@@ -20,11 +20,15 @@
 require 'rspec/rails'
 require 'capybara/cuprite'
 
-Capybara.register_driver(:cuprite) do |app|
+Capybara.register_driver(:better_cuprite) do |app|
+  browser_options = ENV.fetch('CI', nil) ? { 'no-sandbox': nil } : {}
+
   options = {
     window_size: [1280, 1024],
     headless: ENV['CAPYBARA_DEBUG'] != '1',
+    process_timeout: 20,
```

Comment:
I tried this on my other PR and I was still seeing system specs failing intermittently. I wonder if this will actually fix the issue. @smudge

Reply:
ohh, just saw the PR description, I didn't realized there was a collision with the driver. lgtm